bigbluebutton-html5/public/locales/en.json

```
@@ -248,8 +248,13 @@
     "app.presentationUploader.sent": "Sent",
     "app.presentationUploader.exportingTimeout": "The export is taking too long...",
     "app.presentationUploader.export": "Send to chat",
+    "app.presentationUploader.exportAnnotatedPresentation": "Send presentation to chat with annotations",
```

```suggestion
    "app.presentationUploader.exportAnnotatedPresentation": "Send presentation with annotations to chat",
```

---

What do you think about this combined with (long requested) toast notification for everyone when a link is available (the toast to contain the link?

```suggestion
    "app.presentationUploader.exportAnnotatedPresentation": "Send out a download link for the presentation with annotations included"
```